overpy/format/geojson.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+import overpy
+
+
+def dump(result, fp, nodes=False, ways=False, json_args=None):
+    """
+    Use the result from the Overpass API to generate GeoJSON.
+    
+    More information:
+
+    * http://geojson.org/
+
+    :param result: The result from the Overpass API
+    :type result: overpy.Result
+    :param fp: Filepointer to use
+    :param nodes: Export nodes
+    :type nodes: bool
+    :param ways: Export ways
+    :type ways: bool
+    :param json_args: Additional arguments passed to json.dump(...)
+    :type json_args: dict
+    :return:
+    """
+    features = []
+    if nodes:
+        for node in result.nodes:
+            properties = {}
+            features.append({
+                "type": "Feature",
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [
+                        float(node.lon),
+                        float(node.lat)
+                    ]
+                },
+                "properties": properties
+            })
+
+    if ways:
+        for way in result.ways:
+            properties = {}
+            coordinates = []
+            for node in way.nodes:
```

Comment:
Thanks for this excellent PR!
Just one brief suggestion - Some `way`'s have duplicate first and last coordinates and so they ought to be treated as `Polygon` rather than `LineString`

Reply:
Thanks for the information. I will have a look at the documentation.